<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveFormAsTemplate</class>
 <widget class="QDialog" name="SaveFormAsTemplate">
  <property name="windowTitle">
   <string>Save Form As Template</string>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>297</width>
    <height>109</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <layout class="QFormLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="text">
        <string>&amp;Name:</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="buddy">
        <cstring>templateNameEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="templateNameEdit">
       <property name="text">
        <string/>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="minimumSize">
        <size>
         <width>222</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="text">
        <string>&amp;Category:</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="buddy">
        <cstring>categoryCombo</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="categoryCombo"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="horizontalLine">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SaveFormAsTemplate</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>113</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SaveFormAsTemplate</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
